### ISSUE TITLE
Add numeric output block 

### ISSUE THREAD

**Issue by rrd108** [rrd108/vue-mess-detector#313]
this is the current:
```json
  "codeHealthOutput": [
    {
      "info": "Found <bg_err>0 errors</bg_err>, and <bg_warn>30 warnings</bg_warn>, <bg_info>326 lines</bg_info> of code in <bg_info>10 files</bg_info>"
    },
    {
      "info": "Code Health: [<bg_ok>______________________________________________________</bg_ok><bg_warn>______</bg_warn><bg_err></bg_err>] 91%\n"
    },
    {
      "info": "<bg_info>Code health is OK: 91%</bg_info>\n"
    }
  ],
```
Would be cool something like:
```json
  "codeHealth": [
    {
      "errors": 0,
      "warnings": 30,
      "lines": 326,
      "files": 10,
      "code-health": "91%"
    },
  ],
```

**Comment by rrd108** [comment]
https://github.com/brenoepics/vmd-action

**Comment by rrd108** [comment]
implemented in 0.54.0
